src/platform/utilities/date/index.js

```diff
@@ -67,3 +67,14 @@ export function timeFromNow(date, userFromDate = null) {
 
   return 'a moment';
 }
+
+/**
+ * Checks if the passed-in arg is a valid date string, meaning it can be parsed
+ * by Date.parse()
+ *
+ * @param {string} dateString The string to validate
+ * @returns {boolean} If the string is a valid date string
+ */
+export function isValidDateString(dateString) {
+  return !isNaN(Date.parse(dateString));
```

Comment:
Moment will also tell if you if a string is valid: `moment(dateStr).isValid`

Reply:
Ah, I looked for something in lodash but didn't find it and also thought doing the check without relying on a 3rd party dependency was nice.

Reply:
Yeah, I don't think you need to change it, just was pointing it out. In application code that already uses moment, it might be easier to use. But I'd keep this helper moment-free.